#156882158 Landing page with effective UI/UX

chuzksy-codeactive opened the pull request:
#### What does this PR do?
Design landing page for the software product

#### Description of Task to be completed?
Users should see a landing page with effective UI/UX

#### How should this be manually tested?
    Clone the repository
    run npm install
    run npm run start
    open the site on port localhost:4000
#### What are the relevant pivotal tracker stories?
Story ID #156882158

Commit:
chore: add error message

user can see error message on sign up page
show image on sign up page

[Delivers #157283077]

Commit:
feat(landing page design): users should see landing page
- Resign the landing page
- Create css style for the landing page
[Deliver #155882158]

Commit:
feat(landing page design): users should see landing page
- Resign the landing page
- Create css style for the landing page
[Deliver #155882158]

amarachukwu-agbo reviewed (commented):
Good job on the redesigning the landing page for the game, the UI looks so much better. I noticed that we have two files index.html and index2.html which are quite similar. From what I observed, index2.html seems to be the file for the initial landing page while index.html is the file for the new landing page you created. If that is the case, do we still need to keep both files?

Commit:
feat(landing page design): users should see landing page
- Redesign the landing page
Create css style for the landing page
[Deliver #156882158]

emmabaye reviewed (commented):
Good job implementing the landing page and also linking it Angularjs on the frontend.
